src/models/os.ts

```diff
@@ -134,7 +134,7 @@ const archCompatibilityMap: Partial<Dictionary<string[]>> = {
 const tagsToDictionary = (
 	tags: Array<Pick<ResourceTagBase, 'tag_key' | 'value'>>,
 ): Partial<Dictionary<string>> => {
-	const result: Dictionary<string> = {};
+	const result: Dictionary<string> = Object.create(null);
```

Comment:
What is the reason for this?

Reply:
This is a more appropriate way to create a dictionary object since it doesn't include any prototypical chain.
This is more important in cases that you don't trust the values that will be used as keys, since they could be overriding properties defined in the protoypical chain, eg `__proto__`,  `toString` etc, which could be a security issues.
In this case it's not that much of an issue since we do control the hostApp tags, but I decided to go ahead with the change anyway.